pkg/rke/k8s_rke_system_images.go

```diff
@@ -10730,6 +10730,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.8.9
+		"v1.28.15-rancher1-1": {
```

Comment:
could you check if default k8s version need to change?

Reply:
Done